src/Entity/LokaalPersoon.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\LokaalPersoonRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=LokaalPersoonRepository::class)
+ */
+class LokaalPersoon
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
```

Comment:
Please use a UUID instead of a int, as an int may be guessed and could pose a security issue, where UUID is random